lms/static/scripts/frontend_apps/components/LMSFilePicker.tsx

```diff
@@ -75,6 +75,11 @@ type LMSFilePickerProps = {
    * Render path breadcrumbs and allow sub-folder navigation?
    */
   withBreadcrumbs?: boolean;
+
+  /**
+   * An alternative title for the modal dialog. Defaults to "Select file"
```

Comment:
"Alternative title" implies that there is some other "main title" that is shown. I simply change this to "Title for the dialog" or "Custom title for the dialog".